tgui/src/interfaces/clockwork_slab.ract

```diff
@@ -1,5 +1,6 @@
 <ui-display>
   <ui-button action='toggle'>{{data.recollection ? "Recital":"Recollection"}}</ui-button>
+  <ui-button action='component'>Target Component: {{{data.target_comp}}}</ui-button>
```

Comment:
There's no really good way to handle input in tgui, as far as I know; dropdown menus are a fucking myth for liars that don't exist.